src/test/java/com/puppycrawl/tools/checkstyle/internal/XdocsPagesTest.java

```diff
@@ -897,16 +909,33 @@ private static void validatePropertySectionPropertyJavadocTokens(String fileName
         assertEquals("javadoc tokens to check",
                 columns.get(1).getTextContent(), fileName + " section '" + sectionName
                         + "' should have the basic token javadoc description");
-        assertEquals("subset of javadoc tokens "
-                        + CheckUtil.getJavadocTokenText(check.getAcceptableJavadocTokens(),
-                                check.getRequiredJavadocTokens()), columns.get(2).getTextContent()
-                        .replaceAll("\\s+", " ").trim(), fileName + " section '" + sectionName
-                                + "' should have all the acceptable javadoc tokens");
-        assertEquals(
-                CheckUtil.getJavadocTokenText(check.getDefaultJavadocTokens(),
-                        check.getRequiredJavadocTokens()), columns.get(3).getTextContent()
-                        .replaceAll("\\s+", " ").trim(), fileName + " section '" + sectionName
-                                + "' should have all the default javadoc tokens");
+
+        String acceptableTokenText = columns.get(2).getTextContent().trim();
+        final int removePrefixIdx = "subset of javadoc tokens".length();
+        acceptableTokenText = acceptableTokenText.substring(removePrefixIdx);
```

Comment:
Why can't we still verify the `subset of javadoc tokens` text? By splitting the string in this way, that "subset" text will not appear in the assertion failure which lets user know it needs to be there.

Reply:
Initially I  was having some problem with regex matching. I have made the required changes now.